config/7.6.1/dbs/mysql.yaml

```diff
@@ -12,11 +12,11 @@ servers:
               initContainers:
                 - command:
                     [
-                      "sh",
+                      "/bin/bash",
                       "-c",
-                      "until nslookup [[.KieName]]-mysql && nc -vz [[.KieName]]-mysql 3306; do echo waiting for [[.KieName]]-mysql; sleep 2; done;",
+                      "replicas=$(oc get dc ${APPLICATION_NAME}-mysql -o=jsonpath='{.status.availableReplicas}'); until "[" $replicas -gt 0 "]"; do echo waiting for ${APPLICATION_NAME}-mysql; replicas=$(oc get dc ${APPLICATION_NAME}-mysql -o=jsonpath='{.status.availableReplicas}'); sleep 2; done;",
                     ]
-                  image: busybox
+                  image: registry.redhat.io/openshift3/ose-cli
```

Comment:
might want to stick to a tagged version here.

Reply:
tks! done!